libs/utils/utils.js

```diff
@@ -1413,7 +1414,7 @@ export function partition(arr, fn) {
 const preloadBlockResources = (blocks = []) => blocks.map((block) => {
   if (block.classList.contains('hide-block')) return null;
   const { blockPath, hasStyles, name } = getBlockData(block);
-  if (['marquee', 'hero-marquee'].includes(name)) {
+  if (['marquee', 'hero-marquee', 'marquee-container'].includes(name)) {
```

Comment:
Instead of preloading, can't we just not load the block & it's CSS? We can still use it as a marker, but IMO it doesn't make sense to load files for something we don't use

Reply:
We're not using the CSS file right now, but we will get Consonant updates on how a few blocks should look like when inside this combination marquee.

Nevertheless, @zagi25 is looking into @chrischrischris' suggestion of trying to target a section in MEP based on its Section Metadata properties.